quick fixes (#8)
LIT-Protocol/vincentDocs: docs/Developers/Quick-Start.md

	You can follow the [end-to-end code](https://github.com/LIT-Protocol/vincent-dca/tree/main) for the DCA demo.
	:::

In this Quick Start guide you'll learn how to register your first Vincent app:

## 1. Creating Tools & Policies

> 	- **Tools:** Vincent Tools are codified as [Lit Actions](https://developer.litprotocol.com/sdk/serverless-signing/overview) and are used to define the specific operations that your Vincent app can perform on *behalf* of your users. As a Vincent App developer, you can prompt your users to permit these Tools access to their Agent Wallet using the Vincent [Consent Page](../Users/Onboarding.md). This enables your application to perform these specific operations on behalf of the user, fully programmatically according to the guardrails they set.

Reviewer: ```suggestion
	- **Tools:** Vincent Tools are [Lit Actions](https://developer.litprotocol.com/sdk/serverless-signing/overview) that are used to define the specific operations that your Vincent app can perform on *behalf* of your users. As a Vincent App developer, you can prompt your users to permit these Tools access to their Agent Wallet using the Vincent [Consent Page](../Users/Onboarding.md). This enables your application to perform these specific operations on behalf of the user, fully programmatically according to the guardrails they set via Policies.
```

> 	- **Policies:** Users can set guardrails for the Tools they permit which dictate the operating conditions for that Tool. Each Tool can have any number of Policies associated with it, such as maximum daily spending limits or multi-factor authentication (MFA). The Vincent delegation system ensures that all of these policies are met before a given Tool can be executed using the delegated user's Agent Wallet. Just like Tools, Policies are also codified as Lit Actions and your job as the app developer is to provide the the user with a set of optional Policies for each Tool using the Consent Page.

Reviewer: ```suggestion
	- **Policies:** Users can set guardrails for the Tools they permit which dictate the operating conditions for that Tool. Each Tool can have any number of Policies associated with it, such as maximum daily spending limits or multi-factor authentication (MFA). The Vincent delegation system ensures that all of these policies are met before a given Tool can be executed using the delegated user's Agent Wallet. Just like Tools, Policies are also Lit Actions and your job as the app developer is to provide the the user with a set of optional Policies for each Tool using the Consent Page.
```
